pinot-core/src/main/java/org/apache/pinot/core/geospatial/transform/function/ScalarFunctions.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.geospatial.transform.function;
+
+import org.apache.pinot.common.function.annotations.ScalarFunction;
+import org.apache.pinot.core.geospatial.GeometryUtils;
+import org.apache.pinot.core.geospatial.serde.GeometrySerializer;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.io.WKTWriter;
+
+
+/**
+ * Geospatial scalar functions that can be used in transformation.
+ */
+public class ScalarFunctions {
+
+  /**
+   * Creates a point.
+   *
+   * @param longitude longitude
+   * @param latitude latitude
+   * @return the created point
+   */
+  @ScalarFunction
+  public static byte[] stPoint(double longitude, double latitude) {
+    return GeometrySerializer
+        .serialize(GeometryUtils.GEOMETRY_FACTORY.createPoint(new Coordinate(longitude, latitude)));
```

Comment:
(Major) Should this be `GEOGRAPHY_FACTORY` for longitude and latitude?

Reply:
good point. Changed the argument to `x,y`